### DISCUSSION TITLE
ResponseBodyAdvice, 공통 응답 포맷 설정 도입에 관하여

### DISCUSSION THREAD

**Discussion by big-cir** [Attica-org/athens-backend#28] [General]
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/mvc/method/annotation/ResponseBodyAdvice.html

ResponseBodyAdvice 인터페이스를 구현해서 공통 응답 형식을 만들 수 있는것을 봤습니다.
저희 서비스 API 응답 포맷이 대부분 일치합니다. 장/단점을 명확히 하여 도입 해봐도 좋을 것 같습니다.

@Starlight258 @yybmion

**Comment by Starlight258** [comment]
좋습니다 👍 `@ControllerAdvice` +` @ExceptionHandler`를 이용해 전역 예외 처리까지 함께 해도 좋겠네요~
